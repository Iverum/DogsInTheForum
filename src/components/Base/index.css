.Base {
  margin-top: 2.5rem;
}

.Base>header {
  align-items: flex-start;
  border-bottom: grey 1px solid;
  display: flex;
  justify-content: space-between;
}

.Base>header h1 {
  font-size: 3rem;
}

.Base>section {
  margin-top: 2.5rem;
}
