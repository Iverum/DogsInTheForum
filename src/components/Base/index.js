import React, { Component } from 'react'
import './index.css'

export default class Base extends Component {

  render() {
    return (
      <div className="Base row">
        <header className="row">
          <h1>Dogs in the Forum</h1>
        </header>
        <section className="row">
          {this.props.children}
        </section>
      </div>
    )
  }

}
