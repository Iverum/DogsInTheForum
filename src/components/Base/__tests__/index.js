import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Base from '../index'

jest.mock('../../Menu')

test('base is rendered correctly', () => {
  const component = renderer.create(
    <Base />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
