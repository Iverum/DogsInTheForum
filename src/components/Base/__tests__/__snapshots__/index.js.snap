exports[`test base is rendered correctly 1`] = `
<div
  className="Base row">
  <section
    className="row" />
</div>
`;
